repo2docker/buildpacks/julia.py

```diff
@@ -1,18 +1,27 @@
-"""
-Generates a variety of Dockerfiles based on an input matrix
-"""
+"""Generates a Dockerfile based on an input matrix for Julia"""
 import os
 from .conda import CondaBuildPack
 
 
 class JuliaBuildPack(CondaBuildPack):
     """
-    Julia + Conda build pack
+    Julia build pack which uses conda.
+
+    The Julia build pack always uses the parent, `CondaBuildPack`,
+    since Julia does not work with Python virtual environments.
+    See https://github.com/JuliaPy/PyCall.jl/issues/410
 
-    Julia does not work with Virtual Envs,
-    see https://github.com/JuliaPy/PyCall.jl/issues/410
     """
     def get_env(self):
+        """Get additional environment settings for Julia and Jupyter
+
+        Returns:
```

Comment:
I think the docstrings here should explain what exactly is being returned, since the generic docstring for what get_env is returning is already in the base BuildPack. I think explaining each of the env variable set here and why we do it would be the useful thing here.

- JULIA_PATH - base path where all Julia Binaries and libraries will be installed
- JULIA_HOME - path where all Julia Binaries will be installed
- JULIA_PKGDIR - path where all Julia libraries will be installed
- JULIA_VERSION - default version of julia to be installed
- JUPYTER - environment variable required by IJulia to point to `jupyter` executable

Reply:
added 👍